grpc/client_test.go

```diff
@@ -0,0 +1,40 @@
+package grpc
+
+import (
+	"crypto/tls"
+	"testing"
+
+	"github.com/jmhodges/clock"
+	"github.com/letsencrypt/boulder/cmd"
+	"google.golang.org/grpc"
+	_ "google.golang.org/grpc/health"
+)
+
+func TestClientSetup(t *testing.T) {
+	type args struct {
+		c            *cmd.GRPCClientConfig
+		tlsConfig    *tls.Config
+		metrics      clientMetrics
+		clk          clock.Clock
+		interceptors []grpc.UnaryClientInterceptor
```

Comment:
These four fields are identical in all four test cases. Rather than having this args struct, and having to spell out all four of them in every test case, you can just inline the `*cmd.GRPCClientConfig` into the `tests` struct directly:
```
tests := []struct {
  name    string
  cfg     *cmd.GRPCClientConfig
  wantErr bool
}
```
and then have the tests call
```
_, err := ClientSetup(tt.c, &tls.Config{}, clientMetrics{}, clock.NewFake(), []grpc.UnaryClientInterceptor{})
```